dll/win32/comdlg32/lang/cdlg_Ro.rc

```diff
@@ -352,174 +352,174 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
 CAPTION "Imprimare"
 FONT 8, "MS Shell Dlg"
 {
-    DEFPUSHBUTTON   "Con&firmă",     IDOK,     176,164, 50,14, WS_GROUP | BS_DEFPUSHBUTTON
-    PUSHBUTTON      "A&nulează", IDCANCEL, 230,164, 50,14, WS_GROUP
-    PUSHBUTTON      "&Manual…",  pshHelp,  50, 161, 50,14, WS_GROUP
+    DEFPUSHBUTTON   "OK",     IDOK,     176,164, 50,14, WS_GROUP | BS_DEFPUSHBUTTON
+    PUSHBUTTON      "Revocare", IDCANCEL, 230,164, 50,14, WS_GROUP
+    PUSHBUTTON      "&Ajutor",  pshHelp,  50, 161, 50,14, WS_GROUP
 
     GROUPBOX        "Imprimantă",        grp4,   8,  4, 272,84, WS_GROUP
-    CONTROL         "Imp&rimă în fișier", chx1, "Button",BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP,185,36,90,16
-    PUSHBUTTON      "Opți&uni…",    psh2, 212, 17,  60,14, WS_GROUP
-    LTEXT           "N&ume:",         stc6,  16, 20,  36,8
+    CONTROL         "I&mprimare în fişier", chx1, "Button",BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP,185,36,90,16
+    PUSHBUTTON      "Propr&ietăţi…",    psh2, 212, 17,  60,14, WS_GROUP
```

Comment:
&i also here. Please prove with a pic that MS does it like that.

Reply:
Here I made a mistake. It is about "&Preferinţe" and not "Propr&ietăţi…".

![2024-04-30 00_30_39-Resource Hacker - comdlg32 dll mui](https://github.com/reactos/reactos/assets/113553683/fa103479-88e4-496d-83da-991cd2c3312a)

Reply:
Although this is the wrong picture, I will consider this as being resolved now as well, because you proved already (at other locations) that MS decided to use the &i accelerator for that word.

Reply:
I came back with the right one.

![2024-05-05 12_19_02-](https://github.com/reactos/reactos/assets/113553683/e2fc126e-3eea-4185-a386-1ef526f45def)